src/views/StatisticsView.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+    <div>
+        <div class="title">
+            <router-link to="/"><mdicon class="back" name="arrow-left" size="40"/></router-link>
+            <h1>Cobresun Statistics</h1>
+        </div>
+
+        <div>
+            <h2>Most Loved Movie</h2>
+            <p>{{mostLovedMovie}}</p>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+    import { Vue } from 'vue-property-decorator'
+    import axios from 'axios'
+
```

Comment:
Right here you need to add a component decorator (meaning @Component({})) like in the other components